virttest/utils_libvirt/libvirt_secret.py

```diff
@@ -3,12 +3,63 @@
 """
 
 import logging
+import re
+
+from aexpect import remote
+
+from avocado.core import exceptions
 
 from virttest import virsh
+from virttest.utils_test import libvirt
+from virttest.libvirt_xml import secret_xml
 
 LOG = logging.getLogger('avocado.' + __name__)
 
 
+def create_secret(params, remote_args=None):
```

Comment:
Can we directly pass sec_dict?  I think it'll be simpler. What do you think?

Reply:
She has already used sec_dict.

Reply:
params is something like {'sec_dict': {'sec_key1':'value1', 'sec_key2':'value2'}, 'other_key': 'other_value',...} now.
What I mean is that we may pass {'sec_key1':'value1', 'sec_key2':'value2'} directly.

Reply:
Oops, just found that it has been fixed in the latest version.

Reply:
Added sec_dict in  create_secret function as params